build/DownloadSonarPlugin/DownloadAndExtractSonarJS.cs

```diff
@@ -51,10 +51,10 @@ public class DownloadAndExtractSonarJS: Task
         //          package\package.json
 
         // The archive containing the eslintbridge
-        private const string SourceEsLintBridgeFilePattern = "eslint-bridge-*.tgz";
+        private const string SourceEsLintBridgeFilePattern = "sonarjs-*.tgz";
 
         // Sub-folder into which the tar file should be unzipped
-        public const string TargetEslintBridgeFolderName = "eslint-bridge";
+        public const string TargetEslintBridgeFolderName = "sonarjs";
```

Comment:
This is the name of a temporary working folder used when extracting files from the jar.
I didn't really need to rename it, but it made it easy to be sure that we were definitely only getting files from the embedded file (and also easy to compare against the previous version as a sanity check of the number/names of files unzipped).